spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/data/couchbase/CouchbaseReactiveAndImperativeRepositoriesAutoConfigurationTests.java

```diff
@@ -92,7 +92,7 @@ public void shouldCreateInstancesForReactiveAndImperativeRepositories() {
 					CouchbaseReactiveRepositoriesAutoConfiguration.class }) {
 				names.add(type.getName());
 			}
-			return names.toArray(new String[0]);
+			return names.toArray(new String[names.size()]);
```

Comment:
Can't we write it using method reference as return names.toArray(String[]::new) ?

Reply:
No, I don't think so. AFAIK, you can't create an array using the syntax. Even if you could, it would presumably give you a 0-length array which would defeat the purpose of the change that @izeye is proposing.

Reply:
@wilkinsona  You are correct, I mislook it.